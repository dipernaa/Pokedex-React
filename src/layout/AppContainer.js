import React from 'react';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import getMuiTheme from 'material-ui/styles/getMuiTheme';
import AppBar from 'material-ui/AppBar';
import { green500, redA400, white, lightBlack, black } from 'material-ui/styles/colors';

require('../styles/App.scss');

const theme = getMuiTheme({
  palette: {
    primary1Color: green500,
    accent1Color: redA400,
    textColor: lightBlack
  },
  tabs: {
    backgroundColor: white,
    textColor: lightBlack,
    selectedTextColor: black
  },
  toolbar: {
    backgroundColor: white
  },
  inkBar: {
    backgroundColor: green500
  }
});

const AppContainer = (props) => (
  <MuiThemeProvider muiTheme={theme}>
    <div className="app">
      <AppBar showMenuIconButton={false} title="Tacos"/>
      {props.children}
    </div>
  </MuiThemeProvider>
);

export default AppContainer;
