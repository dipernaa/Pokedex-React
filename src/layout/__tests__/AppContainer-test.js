import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AppContainer from '../AppContainer';

const TEST_PROPS = {
  routes: [{
    title: 'test'
  }]
};

it('renders an AppContainer', () => {
  const wrapper = shallow(
    <AppContainer {...TEST_PROPS}>
      <div>HI</div>
    </AppContainer>
  );

  expect(toJson(wrapper.find('.app'))).toMatchSnapshot();
});
