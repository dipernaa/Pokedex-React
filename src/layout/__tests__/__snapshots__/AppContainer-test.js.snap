// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an AppContainer 1`] = `
<div
  className="app"
>
  <AppBar
    title="test"
  />
  <div>
    HI
  </div>
</div>
`;
